migration/20190924-add-externalintegrationslink-table.sql

```diff
@@ -0,0 +1,15 @@
+DO $$
+    BEGIN
+        BEGIN
+            CREATE TABLE externalintegrationslinks (
+                id SERIAL PRIMARY KEY,
+                external_integration_id INTEGER REFERENCES externalintegrations(id),
+                library_id INTEGER REFERENCES libraries(id),
+                other_integration_id INTEGER REFERENCES externalintegrations(id),
+                purpose VARCHAR
+            );
+        EXCEPTION
+            WHEN duplicate_table THEN RAISE NOTICE 'Warning: externalintegrationslinks already exists.';
+        END;
+    END;
+$$;
```

Comment:
I tried this out on my laptop and it looks like this doesn't create the indexes. However, when you're creating a brand new database table you don't need a migration script at all. The database table will be created automatically the first time SQLAlchemy is initialized.

Reply:
I see. So then this isn't really needed at all?

Reply:
Yeah, you can lose it.